cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -114,22 +114,41 @@ def handle_item_deleted(**kwargs):
             gating_api.set_required_content(course_key, module.location, None, None, None)
 
 
-@receiver(GRADING_POLICY_CHANGED)
-@locked(expiry_seconds=GRADING_POLICY_COUNTDOWN_SECONDS, key='course_key')
-def handle_grading_policy_changed(sender, **kwargs):
-    # pylint: disable=unused-argument
-    """
-    Receives signal and kicks off celery task to recalculate grades
-    """
+
+def _enqueue_recompute_grades_task(course_key, grading_policy_hash=None):
     kwargs = {
-        'course_key': six.text_type(kwargs.get('course_key')),
-        'grading_policy_hash': six.text_type(kwargs.get('grading_policy_hash')),
+        'course_key': six.text_type('course_key'),
```

Comment:
There's a merge conflict changing these `six.text_types` to `str()`s. Propose updating all to the `str()` casting.

```suggestion
        'course_key': str('course_key'),
```